### DISCUSSION TITLE
[Feature Request] Modify Discord rich presence text

### DISCUSSION THREAD

**Discussion by loserskater** [ChristopherKlay/StadiaEnhanced#191] [Ideas & Requests]
Instead of 'Playing Stadia' can it be 'Playing {Game}'? I want people to see what game I'm actually playing without having to click my profile and check.

EDIT: Just read the Discord rich presence docs and I see why this isn't possible.

EDIT 2: Looks like StadiaRPC extension does it dynamically. https://github.com/soap-less/StadiaRPC

**Comment by ChristopherKlay** [comment]
StadiaRPC (which isn't supported anymore, since a few months after release of Stadia already), grabs the game title similar to Stadia Enhanced, but it's also only visible if you actually click on the status. The "game" you play stays "Stadia" the whole time, based on the presence being called "Stadia" and this:

`let clientId = "648430151390199818"; // Default Application ID, sets game as "Stadia"`

Changing the actual "game" based on the game you play *would* be possible, but would require a entire app + presence, for every single title, so DiscordRPC could deliver the given infos via the game specific presence to showcase "Playing Destiny 2", for example.

This is sadly a limitation of Discord always showing the presences app name only, with no option to change it dynamically.
